app/data_model/questionnaire_store.py

```diff
@@ -1,56 +1,50 @@
-import simplejson
-
 from app.data_model.answer_store import AnswerStore
 from app.questionnaire.location import Location
+import simplejson as json
 
 
 class QuestionnaireStore:
 
     def __init__(self, storage):
         self._storage = storage
-        self._initial_data = {}
         self.metadata = {}
         self.answer_store = AnswerStore()
         self.completed_blocks = []
 
         raw_data = self._storage.get_user_data()
         if raw_data:
-            self._initial_data = self._deserialise(raw_data)
-            data_copy = self._deserialise(raw_data)
-            self._set_data(data_copy)
-
-    @staticmethod
-    def _deserialise(raw_data):
-        data = simplejson.loads(raw_data)
-        data['COMPLETED_BLOCKS'] = [Location.from_dict(location_dict=completed_block) for completed_block in data['COMPLETED_BLOCKS']]
-        return data
-
-    @staticmethod
-    def _serialise(data):
-        # Override default function to return object as a dict
-        return simplejson.dumps(data, default=lambda o: o.__dict__)
-
-    def _set_data(self, data):
-        self.metadata = data.get('METADATA') or {}
-        self.answer_store.answers = data.get('ANSWERS') or []
-        self.completed_blocks = data.get('COMPLETED_BLOCKS') or []
-
-    def _get_data(self):
-        return {
+            self._deserialise(raw_data)
+
+    def _deserialise(self, data):
+        json_data = json.loads(data)
+        if isinstance(json_data, dict):
```

Comment:
why would this be false and if it is does it indicate an error?

Reply:
Without this pylint throws a `maybe-no-member` error (see https://stackoverflow.com/questions/21545409/pylint-reports-maybe-no-member-error-for-data-loaded-via-simplejson) as there are a few edge cases where the deserialised data may not be a dict e.g. the string "true" would be loaded as `True`. In might be clearer to just disable the rule here.

Reply:
if the `if` stays should there be an `else` to log the error (if it is one?) or throw?